config_schema.json

```diff
@@ -238,7 +238,15 @@
                     "type": "string"
                 },
                 "version": {
-                    "type": "integer"
+                    "oneOf": [
```

Comment:
What's this doing?  Are there cases where the version gets redacted for some reason?

Reply:
Yes there are. Feel free to rip out the "redacted" stuff from the schema. I think it's better to just ignore the red squiggly in the ide when you see redacted, rather than trying to add it to the schema.

Reply:
I'm curious why an integer version number would get redacted.  How is that possibly sensitive?

Reply:
Line 163 of config.py of PMM. Seems like Sohjiro is redacting `version`